docs/reference/announcements-release-notes/880/whats-new-in-88.md

```diff
@@ -118,6 +118,13 @@ The Orchestration Cluster [Identity](/components/identity/identity-introduction.
 With this 8.8 change, the source of truth for Identity and Access Management for the Orchestration Cluster (including Zeebe, Operate, Tasklist, and its APIs) is now the Orchestration Cluster itself. This removes the reliance on the separate [Management Identity](/self-managed/components/management-identity/overview.md) (formerly "Identity") component.
 :::
 
+### Tenant interceptors
+
+In Camunda 8.8, Tenant Interceptors are not supported. If you are using Tenant Interceptors in Camunda 8.7, you must migrate to the new Orchestration Cluster Identity [Tenant management](/components/identity/tenant.md) approach to continue using this feature.
```

Comment:
suggest to rephrase "approach to continue using this feature" since the feature basicalyl does not exist anymore

Reply:
Valid point, wdyt about this?
> you must migrate to the new Orchestration Cluster Identity [Tenant management](/components/identity/tenant.md) approach to use Multi-Tenancy.